Packs/Netmiko/Integrations/Netmiko/Netmiko.py

```diff
@@ -0,0 +1,182 @@
+def cmds_command(client, args):
+
+    # Parse the commands
+    cmds = args.get('cmds')
+    if type(cmds) != list:
+        try:
+            cmds = cmds.split('\n')
+        except Exception as err:
+            return_error(f"The 'cmds' input needs to be a JSON array or carriage return separated text - {err}")
+    cmds[:] = [x for x in cmds if len(x) > 0]
+
+    # Parse the remaining arguments
+    isConfig = True if args.get('isConfig', 'false') == 'true' else False
+    enable = True if args.get('require_enable', 'false') == 'true' else False
+    require_exit = True if args.get('require_exit', 'false') == 'true' else False
+    exit_argument = args.get('exit_argument')
+    raw_print = True if args.get('raw_print', 'false') == 'true' else False
+    disable_context = True if args.get('disable_context', 'false') == 'true' else False
+    override_host = args.get('override_host', None)
+    override_port = args.get('override_port', None)
+    override_platform = args.get('override_platform', None)
+    override_username = args.get('override_username', None)
+    override_password = args.get('override_password', None)
+
+    client.hostname = override_host if override_host else client.hostname
+    client.port = override_port if override_port else client.port
+    client.platform = override_platform if override_platform else client.platform
+    client.username = override_username if override_username else client.username
+    client.password = override_password if override_password else client.password
+
+    # Execute the commands
+    output = client.cmds(require_exit, exit_argument, cmds, enable, isConfig)
+    raw_print_list = list()
+
+    # Output the results
+    if raw_print:
+        md = str()
+        try:
+            for command in output.get('Commands'):
+                raw_print_list.append(command.get('Output'))
+            md = "\n".join(raw_print_list)
+        except Exception as err:
+            md = "Error parsing raw print output"
+            demisto.error(f"Error with raw print output - {err}")
+
+    else:
+        md = tableToMarkdown(f'Command(s) against {client.hostname} ({client.platform}):', output.get('Commands', []))
+    outputs_key_field = None
+    outputs_prefix = None
+    outputs = None
+    if not disable_context:
+        outputs_prefix = "Netmiko"
+        outputs_key_field = 'DateTimeUTC'
+        outputs = output.get('Commands')
+
+    command_results = CommandResults(
+        outputs_prefix=outputs_prefix,
+        outputs_key_field=outputs_key_field,
+        outputs=outputs,
+        readable_output=md
+    )
+    return_results(command_results)
+
+
+def main():
+
+    params = demisto.params()
+    args = demisto.args()
+    command = demisto.command()
+
+    platform = params.get('platform')
+    hostname = params.get('hostname')
+    port = int(params.get('port'))
+    username = params.get('credentials', {}).get('identifier')
+    password = params.get('credentials', {}).get('password')
```

Comment:
```suggestion
    password = params.get('credentials', {}).get('identifier')
```From what you said you changed here is supposed to be an identifier, am I right?

Reply:
No, the change you suggested was to have the sshkey obtained as `params.get('credentials', {}).get('sshkey')` however as per the above format, the sshkey sites under `credentials->credentials` so needs to be `params.get('credentials', {}).get('credentials', {}).get('sshkey')`

Reply:
I could also change the user to be the one listed in credentials->credentials->user, however, it is always the same as credentials->identifier, which is shorter 😄